src/main/resources/application.yml

```diff
@@ -40,3 +40,7 @@ collections:
   monitoring-locations:
     title: NWIS Monitoring Locations
     description: USGS water monitoring locations managed in the National Water Information System
+
+query:
+  limit:
+    max: 10000
```

Comment:
This could just be a constant in the java code. If the goal is to make it configuration, it should be a substitution here with an environment variable to populate it.